code/datums/materials/basemats.dm

```diff
@@ -142,6 +142,17 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	qdel(source.GetComponent(/datum/component/slippery))
 	qdel(source.GetComponent(/datum/component/squeak))
 
+///Quiet and/or hard to see.
+/datum/material/tranquillite
+	name = "tranquillite"
+	id = "tranquillite"
+	desc = "Material with quiet and clear properties"
+	color = "#ffffff" //white
+	categories = list(MAT_CATEGORY_ORE = TRUE, MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/mineral/tranquillite
+	value_per_unit = 0.5
+	beauty_modifier = 0.5
+	armor_modifiers = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
```

Comment:
If you want armor to be unchanged, these should be 1. It is a multiplicative modifier.

Reply:
Err, doesn't that mean Bananium armor currently has 100% resists to most damage types?

Reply:
You got it backwards. thing * 0 = 0.
It does mean bananium armor has *100 bomb and *10 fire armor though, for some reason.

Reply:
Oh, whoops yeah now I get it. And I started a local server and tested it, and it seems that Bananium knight armor is indeed extremely effective bomb armor.

Zero armor is intentional; it's supposed to be a counterpart to bananium (which also has zero armor), and I'll be changing it in future PRs once I start adding uses and sources for Tranquillite.